utils/contract-utils.js

```diff
@@ -14,6 +14,32 @@ const isValidContract = function (contract) {
 	return true;
 };
 
+const isValidLibrary = function (libraries) {
+	if (JSON.stringify(libraries) !== JSON.stringify({})) {
+		return true;
+	}
+
+	return false;
+}
+
+const isValidBytecode = function (libraries, bytecode) {
+	if (isValidLibrary(libraries)) {
```

Comment:
Can you reverse the logic and put it as:
```
if(!isValidLibrary(libraries)) {
return false;
}

// Rest of your code
```

You will add way way more readability like that